corehq/apps/reports_core/static/reports_core/js/charts.js

```diff
@@ -165,7 +165,7 @@ var charts = (function() {
                 }
                 var $svg = d3.select(chartContainer[0]).append("svg");
                 var id = 'chart-' + i;
-                $svg.attr({id: id, width: "50%", height: "200"});
+                $svg.attr({id: id, height: "400"});
```

Comment:
i don't think the `width` attribute actually did anything here. @czue was there something you were trying to accomplish with that?